packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -170,6 +180,22 @@ export class Copy {
         return dsnameIndex !== -1;
     }
 
+    /**
+     * Function that checks if source and target data sets have identical member names
+    */
+    private static async hasIdenticalMemberNames (
+        session: AbstractSession,
+        fromPds: string,
+        toPds: string
+    ): Promise <boolean> {
+        const sourceResponse = await List.allMembers(session, fromPds);
```

Comment:
I do like how clean this implementation looks! 🙏 
And I know this might be a bit picky, but in order to minimize the number of calls to the mainframe, could we cache this call? 
`await List.allMembers(session, fromPds);`

or perhaps return it as part of the function too?  😅 

That way, the `copyPDS doesn't have to make the same call to get the list of members from the `fromPds` 😋